internal/pkg/api/hardware.go

```diff
@@ -3,32 +3,26 @@ package api
 import (
 	"fmt"
 	"os"
+	"path/filepath"
 
+	"github.com/aws/eks-anywhere/pkg/providers/tinkerbell/hardware"
 	"github.com/gocarina/gocsv"
 )
 
 const (
-	Dell       = "dell"
-	HP         = "hp"
-	SuperMicro = "supermicro"
+	HardwareVendorDell       = "dell"
+	HardwareVendorHP         = "hp"
+	HardwareVendorSuperMicro = "supermicro"
+	HardwareVendorAgnostic   = "agnostic"
```

Comment:
You might want to use 'unspecified'. 'Agnostic' suggests it should 'just work' and that the underlying technology is irrelevant. I'm not sure we  know that hence it might be safer to use 'unspecified' which doesn't make any propositions.